sdk/cosmosdb/cosmos/src/common/constants.ts

```diff
@@ -218,6 +218,12 @@ export const Constants = {
     min: "min"
   },
 
+  // Deprecated due to typo, use EffectivePartitionKeyConstants instead.
```

Comment:
I believe if you do a doc comment and use `@deprecated`, then TS will strike through the api, e.g.

![image](https://user-images.githubusercontent.com/639216/124828142-ba78d180-df2b-11eb-8174-b17f58821ce3.png)

https://api-extractor.com/pages/tsdoc/tag_deprecated/

Reply:
Yeah, I originally went for that but it doesn't seem to do anything inside of the object. Is it a class specific modifier? I noticed other comments (which are also scrubbed from the public API doc) which mention deprecation https://github.com/Azure/azure-sdk-for-js/blob/82c2ac2475722ba1bfaa8e5230a2ccb7d8b796cc/sdk/cosmosdb/cosmos/src/common/constants.ts#L88